### PR TITLE
Add Western Sahara compatibility

### PR DESCRIPTION
**Pull request by Dahlgren**
**When merged this pull request will:**
- Add support for Western Sahara terrain Sefrou-Ramal

### CODE REVIEW

File: optionals/sys_ws/config.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        requiredAddons[] = {"acre_main"};
+        author = ECSTRING(main,Author);
+        authors[] = {"Rotators Collective"};
+        url = ECSTRING(main,URL);
+        VERSION_CONFIG;
+    };
+};
+
+class CfgAcreWorlds {
+    class stozec {
```
Comment (jonpas) [inline optionals/sys_ws/config.cpp]: This doesn't seem correct.

Comment (Dahlgren) [reply optionals/sys_ws/config.cpp]: Good eyes, fixed!

**Review by jonpas — APPROVED** [review]

File: optionals/sys_ws/LICENCE [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Some items in this component (this folder and all subfolders)
```
Comment (BaerMitUmlaut) [inline optionals/sys_ws/LICENCE]: Typo in filename.

**Review by BaerMitUmlaut — CHANGES_REQUESTED** [review]

File: optionals/sys_ws/LICENCE (hunk 2 above)
Comment (jonpas) [reply optionals/sys_ws/LICENCE]: In CSLA too actually, nice catch!

Comment (Dahlgren) [reply optionals/sys_ws/LICENCE]: Fixed 🙂

**Pull request merged**